dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM python:3.10-slim AS base
+
+ENV LANG C.UTF-8
+ENV PYTHONUNBUFFERED 1
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -y --no-install-recommends \
+    build-essential \
+    tzdata \
+    git \
+    libatlas-base-dev\
+    && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* && \
+    pip install --no-cache-dir poetry==1.4.0 setuptools && poetry config virtualenvs.create false
```

Comment:
Any reason to use poetry==1.4.0 instead of a newer one?
I think it's good to lock it, but I wonder if we can use the latest version.

Reply:
No reason at all, just forgot to bump the version. Nice catch!
Changed it to 1.8.3.